src/write.rs

```diff
@@ -674,16 +723,7 @@ impl<S: Span> Report<'_, S> {
                             let underline = get_underline(col).filter(|_| row == 0);
                             let [c, tail] = if let Some(vbar_ll) = vbar {
                                 let [c, tail] = if underline.is_some() {
-                                    // TODO: Is this good?
-                                    if ExactSizeIterator::len(&vbar_ll.label.char_span) <= 1 || true {
-                                        [draw.underbar, draw.underline]
-                                    } else if line.offset() + col == vbar_ll.label.char_span.start {
-                                        [draw.ltop, draw.underbar]
-                                    } else if line.offset() + col == vbar_ll.label.last_offset() {
-                                        [draw.rtop, draw.underbar]
-                                    } else {
-                                        [draw.underbar, draw.underline]
-                                    }
```

Comment:
Clippy said this logic is a bit buggy is true. The first `if` condition will always evaluate to `true` so really the whole `if` expression can be replaced by the body in the first branch.

Reply:
It's not buggy, but it is correct: the `|| true` is there because it's temporarily disabling a feature that might be reenabled later. If you remove the `|| true` you'll be able to see the change in the output!

Reply:
Added the code back with a comment :)